sql/step_04_all_following_steps.sql

```diff
@@ -71,7 +71,7 @@ source step_74_operation_report_button.sql;
 source step_75_add_prefix_to_table_names.sql;
 source step_76_dicom_button_in_admission_patient.sql;
 source step_77_laboratory_button_in_admission_patient.sql;
-source step_78_drop_lab_exam_date.sql;
+source step_78_logout_menu_item.sql;
```

Comment:
You can not replace step_78; that sql file is needed;  the next available step number is 82.